src/wasm/wasm-type.cpp

```diff
@@ -338,6 +344,9 @@ HeapTypeInfo::HeapTypeInfo(const HeapTypeInfo& other) {
 
 HeapTypeInfo::~HeapTypeInfo() {
   switch (kind) {
+    case BasicKind:
+      basic.HeapType::~BasicHeapType();
```

Comment:
Does it not want to you write `basic.~HeapType::BasicHeapType()`?   Would that work?

Reply:
No, unfortunately that doesn't compile at all. I just ended up getting rid of this destructor call, since it wasn't doing anything anyway 🤷‍♂️